src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -296,6 +362,216 @@ const formConfig = {
+                treatments: {
+                  'ui:options': {
+                    itemName: 'Record',
+                    viewField: () => {
+                      return (<div>hello</div>);
+                    }
+                  },
+                  items: {
+                    treatment: {
+                      'ui:order': ['treatmentCenterName', 'privateMedicalRecordsReleaseAccepted', 'view:privateMedicalRecordsReleasePermissionRestricted', 'startTreatment', 'endTreatment', 'treatmentCenterCountry', 'treatmentCenterStreet1', 'treatmentCenterStreet2', 'treatmentCenterCity', 'treatmentCenterState', 'treatmentCenterPostalCode', 'treatmentCenterPhone'],
+                      treatmentCenterName: { // TODO: is this required?
+                        'ui:title': 'Name of private provider or hospital'
+                      },
+                      privateMedicalRecordsReleaseAccepted: {
+                        'ui:title': 'I give my consent, or permission, to my doctor to only release records related to this condition'
+                      },
+                      'view:privateMedicalRecordsReleasePermissionRestricted': {
+                        'ui:description': () => {
+                          return (<div className="usa-alert usa-alert-warning no-background-image">
+                            <span>Limiting consent means that your doctor can only share records that are directly related to your condition. This could add to the time it takes to get your private medical records.</span>
+                          </div>);
+                        },
+                        'ui:options': {
+                          expandUnder: 'privateMedicalRecordsReleaseAccepted' // TODO: prevent auto
+                        }
+                      },
+                      startTreatment: {
+                        'ui:widget': 'date',
+                        'ui:title': 'Approximate date of first treatment since you received your rating'
+                      },
+                      endTreatment: {
+                        'ui:widget': 'date',
+                        'ui:title': 'Approximate date of last treatment'
+                      },
+                      treatmentCenterCountry: { // TODO: need to restrict these via select
+                        'ui:title': 'Country'
+                      },
+                      treatmentCenterStreet1: {
+                        'ui:title': 'Street'
+                      },
+                      treatmentCenterStreet2: {
+                        'ui:title': 'Street'
+                      },
+                      treatmentCenterCity: {
+                        'ui:title': 'City'
+                      },
+                      treatmentCenterState: {
+                        'ui:title': 'State'
+                      },
+                      treatmentCenterPostalCode: {  // TODO: need to validate this
```

Comment:
Yeah, I think we keep the comments but hold off on validation until we make sure we know what EVSS is going to want. Hopefully it's just our standard validation. :crossed_fingers: